invenio_communities/assets/semantic-ui/js/invenio_communities/organizations/OrganizationsList.js

```diff
@@ -1,6 +1,14 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2024 CERN.
+// Copyright (C) 2024 KTH Royal Institute of Technology.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
 import React from "react";
 import PropTypes from "prop-types";
 import { Popup } from "semantic-ui-react";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
```

Comment:
that is wrong. it should be the link from invenio_communities
`import { i18next } from "@translations/invenio_communities/i18next";`